less/about/preferences.less

```diff
@@ -272,105 +272,6 @@ table.sortableTable {
   color: @braveOrange;
 }
 
-.prefTabContainer .switchControl {
-  display: inline-block;
-  vertical-align: middle;
-  align-items: center;
-  padding: 5px;
-  cursor: pointer;
-
-  &.disabled {
-    .switchBackground {
-      opacity: 0.3;
-    }
-    .switchControlRightText, .switchControlLeftText {
-      opacity: 0.3;
-    }
-    cursor: default;
-  }
-
-  &:not(.disabled) {
-    .switchBackground.switchedOn {
-      background-color: @switchBG_on;
-      .switchIndicator {
-        right: 2px;
-      }
-    }
-  }
-
-  &.large {
-    .switchBackground {
-      height: 26px;
-      width: 60px;
-      .switchIndicator {
-        height: 22px;
-        width: 22px;
-      }
-    }
-  }
-
-  &.small {
-    .switchBackground {
-      height: @smallSwitchHeight;
-      width: @smallSwitchWidth;
-      .switchIndicator {
-        height: @smallSwitchNubDiameter;
-        width: @smallSwitchNubDiameter;
-        top: 1px;
-        right: calc(@smallSwitchWidth - @smallSwitchNubDiameter - @switchNubRightMargin);
-      }
-    }
-  }
-
-  > label {
-    vertical-align: middle;
-  }
-
-  .switchControlText {
-    margin: auto 0;
-  }
-
-  .switchControlLeftText {
-    margin: auto 5px auto 0;
-  }
-
-  .switchControlRightText {
-    margin: auto 0 auto 5px;
-  }
-
-  .switchControlTopText {
-    margin: 0 auto 5px auto;
-  }
-
-  .switchControlRight {
-    display: flex;
-  }
-
-  .switchMiddle {
-    display: flex;
-    flex-direction: column;
-  }
-
-  .switchBackground {
-    background-color: @switchBG_off;
-    border-radius: 12px;
-    height: @switchHeight;
-    position: relative;
-    width: @switchWidth;
-    box-shadow: @switchShadow;
-
-    .switchIndicator {
-      background-color: white;
-      border-radius: 100%;
-      height: @switchNubDiameter;
-      position: absolute;
-      top: @switchNubTopMargin;
-      width: @switchNubDiameter;
-      box-shadow: @switchNubShadow;
-    }
-  }
-}
-
```

Comment:
Previously the text was rendered with a label element outside the `SwitchControl`. This made it easier to have custom spacing without affecting the switchControl css, but it broke the label being able to toggle the switch. Now that the `SettingsCheckbox` (and all usages of that) is passing the text to `SwitchControl` for labels,  the style would have to either be part of `SwitchControl` or a style override from the usage location, adding to the fragmented style definition problem for the component, that will have to be resolved or integrated in a future refactor. Note that not all uses of `SettingsCheckbox` have this spacing (see the Payments tab), so this spacing is very specific to a couple of places. Happy to do that if it can't be resolved now, but thought I'd try just in case it was a simple ok in order to avoid the fragmentation.